samples/keys/hw_unique_key/src/derive_key_tfm.c

```diff
@@ -26,7 +26,12 @@ psa_status_t derive_key(psa_key_attributes_t *attributes, uint8_t *key_label,
 
 	*key_id_out = PSA_KEY_ID_NULL;
 
+#if CONFIG_CRACEN_HW_PRESENT
+	/* On platforms with Cracen, the HUK is an AES key. */
+	status = psa_key_derivation_setup(&op, PSA_ALG_SP800_108_COUNTER_CMAC);
+#else
 	status = psa_key_derivation_setup(&op, PSA_ALG_HKDF(PSA_ALG_SHA_256));
+#endif
```

Comment:
Let's move the ALG into a define at file level scope so we don't have an ugly ifdef in the code.

Same for the label.